### PR TITLE
feat: adds state audits & audit dashboard state filtering

### PR DESCRIPTION
**Pull request by mrferris**
This PR:

- activates population of state root data for blocks 0 - 1 million via a new `glados_monitor_state` container in `docker-compose.yml`
- activates auditing of state root data via a new `glados_audit_state` container in `docker-compose.yml`
- implements toggling of the audit dashboard between history and state

### CODE REVIEW

**Review by pipermerriam — COMMENTED** [review]
:rocket:

File: docker-compose.yml [hunk 1]
```diff
@@ -49,6 +63,18 @@ services:
...
+    image: portalnetwork/glados-monitor:latest
+    environment:
+      RUST_LOG: warn,glados_monitor=info
+    depends_on:
+      - glados_audit
+      - glados_postgres
+    networks:
+      - glados-net
+    restart: always
+
```
Comment (KolbyML) [inline docker-compose.yml:77]: We don't support latest state yet, this should be set to import the state_roots from 0 to 1 million

```suggestion
  glados_monitor_state:
    command: "--database-url postgres://postgres:${GLADOS_POSTGRES_PASSWORD}@glados_postgres:5432/glados populate-state-roots-range --start-block-number 0 --end-block-number 1000001 --provider-url ${GLADOS_PROVIDER_URL?Glados monitor provider URL required}"
    image: portalnetwork/glados-monitor:latest
    environment:
      RUST_LOG: warn,glados_monitor=info
    depends_on:
      - glados_audit
      - glados_postgres
    networks:
      - glados-net
    restart: always
```

Without this change the audits will just fail. We will enable `follow-head-state` after we have bridges gossiping latest

**Review by KolbyML — COMMENTED** [review]
We can't enable `follow-head-state` yet as we don't gossip latest state data onto the network yet. This will cause all audits to fail. Instead we need to use `populate-state-roots-range --start-block-number 0 --end-block-number 1000001` which will make it so state audit only audits state we are actively gossiping on the network

**Review by KolbyML — APPROVED** [review]
:shipit:

**Pull request merged**